contracts/strategies/stkCvxFXS/stkCvxFxsVault.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "../../GenericVault.sol";
+
+struct EarnedData {
```

Comment:
Consider adding this to the `ICvxFxsStaking` interface.

```
interface ICvxFxsStaking {
    struct EarnedData {
        address token;
        uint256 amount;
    }

    function claimableRewards(
        address _account
    ) external view returns (EarnedData[] memory userRewards);
}

...

function claimableRewards() external view returns (ICvxFxsStaking.EarnedData[] memory) {
```

Reply:
Also, `claimableRewards` doesn't seem to be called by any of the other contracts - is it necessary to have it as part of this contract?

Reply:
> Also, `claimableRewards` doesn't seem to be called by any of the other contracts - is it necessary to have it as part of this contract?

it's just a utility view, someone requested as it makes it slightly easier to calc rewards for harvesting (as opposed to querying staking contract directly)